das-dashboard/backend/shared/builders/atom_db_builder.py

```
@@ -0,0 +1,346 @@
+        _require(source, *self._MORK_CONNECTION_FIELDS, *self._MONGO_CONNECTION_FIELDS, label=label)
+        self._validate_cluster_nodes(source, "mongo_cluster", "mongo_nodes", label)
+
+        mongodb = {
+            "image": self.MONGO_IMAGE,
+            "endpoint": self._endpoint(
+                _get(source, "mongo_endpoint"),
+                _get(source, "mongo_port"),
+            ),
+            "username": _get(source, "mongo_username"),
+            "password": _get(source, "mongo_password"),
+            "cluster": _get(source, "mongo_cluster", False),
```

Ruff (0.15.18)

[warning] Boolean positional value in function call

(FBT003)